app/components/common/candidate/candidate_page.html

```diff
@@ -39,6 +39,13 @@
       <div class="component__heading" ng-repeat-start="(label, amount) in $ctrl.contributions_by_type_percentages">{{ label }}</div>
       <odca-money-bar-chart value="{{ amount }}" max="1000" precision="1" format="percentage" color="green" ng-repeat-end></odca-money-bar-chart>
     </div>
+    <div class="see-more" ng-if="!!$ctrl.supporting.total_contributions">
+      <a class="see-more__content" ui-sref="appMain.committee.main({filer_id: $ctrl.candidate.filer_id})")"><span class="text-no-break">See all contributions<span class="fa fa-chevron-right"></span></span></a>
```

Comment:
It's okay for the "See all contributions" to wrap, as long as the chevron doesn't become a widow. I would move the `no-text-break` to
```html
See all <span class="text-no-break">contributions<span class="fa fa-chevron-right"></span></span>
```